lib/fog/aws/models/compute/vpc.rb

```diff
@@ -11,6 +11,7 @@ class VPC < Fog::Model
         attribute :tenancy,          :aliases => 'instanceTenancy'
         attribute :is_default,       :aliases => 'isDefault'
         attribute :ipv_6_cidr_block_association_set,  :aliases => 'ipv6CidrBlockAssociationSet'
+        attribute :cidrBlockAssociationSet,  :aliases => 'cidrBlockAssociationSet'
```

Comment:
For consistency I think we probably want the attribute name to be underscore separated instead of camelcase?